uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/layer/impl/GridLienzoPanel.java

```diff
@@ -123,7 +167,7 @@ void addMouseUpHandler() {
         addMouseUpHandler((e) -> refreshScrollPosition());
     }
 
-    private void setupDefaultHandlers() {
+    void setupDefaultHandlers() {
```

Comment:
Why not `protected`?

Reply:
This class contains lots of default access level methods. It is a favourite approach by @karreiro for unit testing where a method needs mocking but not part of the _public_ API. I took this opportunity to make this method the same access level as other _helper_ methods on the class.. I see @danielezonca is an advocate of not using this access level. I see @karreiro is an advocate... There's only one way to resolve this...... FIGHT!

No, seriously, I'm open to understanding both of your view points. I think IF a method is considered something that is considered purely a 100% internal implementation detail, default access has it's merits. However who is to know what methods a sub-class developer might like/need to override and hence use of `protected` makes sense.....

Reply:
@manstis @danielezonca @karreiro 
I know Guillerme is a "default" supporter. 
Just to learn from mistakes, though, I would like to remember that when the "AbstractPopover" has been implemented, I discussed almost an hour to try to convince to use "protected" modifiers - I've been un-heard. And the first thing I had to do to re-use it in scesim has been to rewrite it because otherwise it was not re-usable.
It is extremely hard to foresee what would be the needs of a specific component, and kie-common codebase is full of widget absolutely un-reusable exactly for that reason.
If a developer "broke" a functionality overriding a method, it is his/her responsability - and this is possible regardless of the modifier applied.
In that specific case, it is almost sure that, sooner or later, someone will need to override the "default handlers" for some reason - and that should not need a widget rewrite.
A good code is something that is easy to extend/reuse, not something that works only in the specific situation where it has been firstly used.

So, let's be "pragmatic" and stop shooting on our own feet :) :) :)

Reply:
Updated _default_ access to `protected` in this class.

Reply:
- and I can adopt `protected` over _default_ for the reasons given ongoing.